ldk/go/examples/search-searchbar/loop/loop.go

```diff
@@ -0,0 +1,83 @@
+package loop
+
+import (
+	"context"
+
+	ldk "github.com/open-olive/loop-development-kit/ldk/go"
+)
+
+const (
+	backgroundColor = "#fff"
+	highlightColor  = "#651fff"
+	primaryColor    = "#666"
+)
+
+// Serve creates the new loop and tells the LDK to serve it
+func Serve() error {
+	l := ldk.NewLogger("loop-example")
```

Comment:
I think this should be more specific. 
`ldk.NewLogger("example-search-searchbar")`
or something like that.

Reply:
Changing that gives me pause as it's "loop-example" in all the examples right now, and we'd need to change it everywhere plus remember to change it when creating a new example (if we copy an existing one). Thoughts?

Reply:
That's a good point. I think it should be specific everywhere. We have missed it in all the other examples. You're right that it will make copy pasting more intensive, but I think it is worth the cost to have logs accurately labeled.

Consider the situation where you might have an example build from something you worked on in the past. You go to deploy a new example for testing and see a log. You assume the log is from your new example because you forgot you had the other example already build. Could mislead a person into following the wrong path.

Reply:
Fair enough counterpoint. That could be confusing in the logs. I'll change this and try to change it everywhere else to be specific as well. 👍